test/tests.py

```diff
@@ -134,10 +134,25 @@ def _():
     succeed("su operator -c 'liquidswap-cli --help'")
 
 
+# backup requires manual setup, so we skip it in this test
```

Comment:
AFAICT backup plugin is no longer available

Reply:
This is also an oversight.  We are no longer shipping the backup plugin anyway.